components/EventCard/styles.ts

```diff
@@ -1,28 +1,38 @@
 
 import { StyleSheet } from 'react-native'
-import { lightGray, peach, shadowColor} from '../../assets/colors/colors'
+import { lightGray, lightPeach, peach, shadowColor} from '../../assets/colors/colors'
 
 export const styles = StyleSheet.create({
   cardContainer: {
-    backgroundColor: peach,
+    backgroundColor: lightPeach,
+    borderColor: peach,
     borderRadius: 10,
-    elevation: 5,
-    margin: 10,
-    overflow: 'hidden',
+    borderWidth: 2,
+    flex: 1,
+    height: 202,
+    margin: 5,
+    overflow: 'scroll',
```

Comment:
should not use this, instead use `hidden` to cover overflowing layout.
```suggestion
    overflow: 'hidden',
```